seed/static/seed/js/controllers/inventory_detail_controller.js

```diff
@@ -475,7 +476,7 @@ angular.module('BE.seed.controller.inventory_detail', []).controller('inventory_
      * save_item: saves the user's changes to the Property/TaxLot State object.
      */
     const save_item_resolve = (data) => {
-      $scope.$emit('finished_saving');
```

Comment:
This emit statement looks like it was related to hiding the form, but I went back several years and couldnt find any reference to `'finished_saving'` outside of these two `$emits`

Honestly, I'm not sure how this worked before.